src/common.props

```diff
@@ -15,10 +15,10 @@
     <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
     <NBenchVersion>1.2.2</NBenchVersion>
     <ProtobufVersion>3.10.0</ProtobufVersion>
-    <NetCoreTestVersion>netcoreapp2.1</NetCoreTestVersion>
+    <NetCoreTestVersion>netcoreapp3.0</NetCoreTestVersion>
```

Comment:
Hold off on this if you can - requires a much bigger set of changes on the build system side to support it.